code/modules/jobs/job_types/roguetown/youngfolk/apothecary.dm

```diff
@@ -36,15 +36,16 @@
 	)
 
 	subclass_skills = list(
-		/datum/skill/combat/knives = SKILL_LEVEL_APPRENTICE,
+		/datum/skill/combat/knives = SKILL_LEVEL_JOURNEYMAN,
```

Comment:
Doesn't do anything, functionally. Apothecaries already can't really capitalize on their knife skills because they lack any meaningful armor and any meaningful mitigation traits.